examples/resize/resize.go

```diff
@@ -0,0 +1,66 @@
+// goncurses - ncurses library for Go.
+
+/* This example demonstrates the ability to resize. Only one of detecting SIGWINCH or KEY_RESIZE
+ * is strictly needed, but depending on the options ncurses was built with, one or the other may
+ * work better. */
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	gc "github.com/rthornton128/goncurses"
+)
+
+var stdscr *gc.Window
+
+func main() {
+	start := time.Now()
+	sigWinChCount := 0
+	keyResizeCount := 0
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGWINCH)
+
+	stdscr, _ = gc.Init()
+	stdscr.Timeout(0)
+	defer gc.End()
+
+	for {
+		select {
+		case <-sigs:
+			sigWinChCount++
+			resize()
+		default:
+			c := stdscr.GetChar()
+			if c == 'q' {
+				return
+			} else if c == gc.KEY_RESIZE {
+				keyResizeCount++
+				//resize()
+			}
```

Comment:
`else if` is almost always a bad idea. In this case, the guard clause makes it pointless. Consider a `switch` statement or removing the else.

```
if c == 'q' {
  return
}

if c == gc.KEY_RESIZE {
  keyResizeCount++
  //resize()
}

Reply:
Changed to `switch`